### PR TITLE
docs: Add filetop guide

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
This guide is rather simple as there is a single pod performing a very simple operation. I tried some of the examples available here (https://github.com/kubernetes/examples) but they are a bit complex to install and they don't produce that much I/O, hence I decided to use this approach. 

Fixes https://github.com/kinvolk/inspektor-gadget/issues/477

### CODE REVIEW

File: docs/guides/filetop.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ubuntu-hirsute   default          mypod            mypod            642727  apt-get          73     0      4568    0       R security.ubuntu.com_ubuntu_dists_focal-security_restricted_binary-amd64_Packages.lz4
+ubuntu-hirsute   default          mypod            mypod            642727  apt-get          70     0      4435    0       R security.ubuntu.com_ubuntu_dists_focal-security_universe_binary-amd64_Packages.lz4
+ubuntu-hirsute   default          mypod            mypod            642727  apt-get          19     0      1172    0       R archive.ubuntu.com_ubuntu_dists_focal_multiverse_binary-amd64_Packages.lz4
+```
+
+After the initial installation is done, we can see how git uses a temporary file to store the repository being cloned:
+
+```
+NODE             NAMESPACE        POD              CONTAINER        PID     COMM             READS  WRITES R_Kb    W_Kb    T FILE
+ubuntu-hirsute   default          mypod            mypod            647042  git              0      1070   0       4280    R tmp_pack_2rpZd
```
Comment (alban) [inline docs/guides/filetop.md:44]: Could you explain what the "T" column means?

**Review by alban — APPROVED** [review]
Thanks! One question about the columns, but LGTM otherwise. Is it allowed to use filetop on several nodes?

File: docs/guides/filetop.md (hunk 1 above)
Comment (mauriciovasquezbernal) [reply docs/guides/filetop.md:53]: Sure. I added a sentence (copied form the BCC repo https://github.com/iovisor/bcc/blob/master/tools/filetop_example.txt) indicating it. I also documented a bit more the flags and arguments that the tool accepts.

**Review by blanquicet — APPROVED** [review]
LGTM

**Pull request merged**